src/condor_startd.V6/boinc_mgr.cpp

```diff
@@ -445,10 +441,8 @@ bool
 BOINC_BackfillMgr::walk( BoincSlotMember member_func )
 {
```

Comment:
Does anyone call BOINC_BackfillMgr::walk?  If not, can we just remove it?

Reply:
Sounds good to me.